tests/unittests/utils/test_depth_anything.py

```
@@ -228,3 +245,396 @@ def test_none_frame_size_skips_resize(self):
+        )
+
+        assert sample["da3_export_path"] == os.path.join(
+            export_dir, "sample", "gs_video", "0000_wander.mp4"
+        )
+
+
+class TestDepthAnythingV3InferenceForwarding:
+    """The config reaches self._model.inference, not just the config object."""
+
+    @staticmethod
+    def _make_model(config_dict, scale_factor=None):
```

Ruff (0.16.1)

[warning] Missing return type annotation for staticmethod `_make_model`

(ANN205)